quickstart/docker/image/Dockerfile

```diff
@@ -19,6 +18,7 @@ RUN apt-get update \
     #   just build the Dockerfile \
     # to use a specific version of ziti, specify ZITI_VERSION_OVERRIDE then  \
     #   build the Dockerfile
+COPY . /docker.build.context
```

Comment:
after APT so we don't have to wait for it every time the build context changes when iterating locally